changes/ticket29662

```diff
@@ -0,0 +1,5 @@
+  o Minor features (debugging):
+    - Introduce tor_assertf() and tor_assertf_nonfatal() to enable logging of
+      additional information during assert failure. Now we can use format
+      strings to include pieces of information that is relevant for trouble
```

Comment:
Grammar nitpick:
```suggestion
      strings to include pieces of information that are relevant for trouble
```

Reply:
Fixed in 34c6eb25d88ad214ca5412b04573a3a1f5404de8.